sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataReference.cs

```diff
@@ -77,11 +77,11 @@ internal static class NuGetMetadataReference
         public static IEnumerable<MetadataReference> MicrosoftAspNetCoreHostingAbstractions(string packageVersion) =>
             NugetMetadataFactory.Create("Microsoft.AspNetCore.Hosting.Abstractions", packageVersion);
 
-        public static IEnumerable<MetadataReference> MicrosoftAspNetCoreHttpAbstractions(string packageVersion) =>
+        public static IEnumerable<MetadataReference> MicrosoftAspNetCoreHttpAbstractions(string packageVersion = "2.2.0") =>
             NugetMetadataFactory.Create("Microsoft.AspNetCore.Http.Abstractions", packageVersion);
 
         public static IEnumerable<MetadataReference> MicrosoftAspNetCoreHttpFeatures(string packageVersion) =>
-            NugetMetadataFactory.Create("Microsoft.AspNetCore.Http.Features", packageVersion);
+            NugetMetadataFactory.CreateWithCommandLine("Microsoft.AspNetCore.Http.Features", packageVersion);
```

Comment:
I had to use `CreateWithCommandLine` since `Create` no longer resolved the reference on my local environment.